ballerina/soap.bal

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/mime;
+
+# Soap version.
```

Comment:
Shall we update doc comments properly. Lets follow HTTP standards [1] 

[1] - https://github.com/ballerina-platform/module-ballerina-http/blob/0c9cad9390e648022e63da01420e749ccc02b1ed/ballerina/http_constants.bal#L20

Reply:
Updated in [b0a493b](https://github.com/ballerina-platform/module-ballerina-soap/pull/82/commits/b0a493b6521e16253fcd1624efdd719fe1f907dc)